### PR TITLE
refactored code to make it more readable and manageable.

### PR DESCRIPTION
**Pull request by nisarhassan12**
image

### CODE REVIEW

File: src/components/index/Promo.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            left: 0;
+            width: 100%;
+            height: 100%;
+            background: #8c9bae;
+            z-index: 1000;
+            opacity: .1;
+        }
+    }
+
+    .promo__video {
```
Comment (JesterOrNot) [inline src/components/index/Promo.js]: Why are there 2 underscores here?

Comment (nisarhassan12) [reply src/components/index/Promo.js]: Thanks! :slightly_smiling_face: This is part of BEM a naming convention for naming classes in CSS. You can find out more about it here http://getbem.com/

Comment (JesterOrNot) [reply src/components/index/Promo.js]: Sorry I'm still learning CSS :sweat_smile:

Comment (nisarhassan12) [reply src/components/index/Promo.js]: Thanks! No Problem.

**Pull request merged**